src/components/icon/icons/co2.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Co2({
+  stroke,
+  fill,
+  strokeWidth,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg style={style} {...rest}>
```

Comment:
There should be a better way to do this in the future instead of 35 copy pasted lines of code

Reply:
Ok :)